EXT_SpeciesType.md

```diff
@@ -0,0 +1,80 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+openPMD extension ID: `4`
+
+
+Introduction
+------------
+
+This convention is for standardizing the names of particle species, e.g. in
+particle physics.
+
+
+Additional Record Attribute
+---------------------------
+
+The following additional attribute for openPMD `records` is defined in this
+extension:
+
+- `SpeciesType`
+  - type: *(string)*
+  - scope: *recommended*
+  - description: particle species in this record, semicolon-separated list
```

Comment:
Replace "semicolon-separated list" with "If there are multiple species to be specified, they can be specified using a semicolon separated list."